src/Matches.js

```diff
@@ -82,42 +82,49 @@ class Matches extends React.Component {
         
     }
 
-
     render(){
         let matchingSection;
         if (this.state.matchesExist == "exists"){
             let userItems = [];
             for (var i = 0; i < this.state.userIds.length; i++){
                 var pos_user = this.state.userIds[i];
                 userItems.push(
-                    <button className='pos-user' key={pos_user} value = {this.state.userIds[i]+"|"+this.state.firstnames[i]} onClick = {this.selectUser}>
+                    <button className='pos-user' 
+                            key={pos_user} 
+                            value = {this.state.userIds[i]+"|"+this.state.firstnames[i]} 
+                            onClick = {this.selectUser}>
                         {this.state.firstnames[i]}
-                    </button>
+                    </button>  
                 )
+                
+
             }
-            matchingSection = 
-                <div>
-                    <td id='Matches-congrats'>
-                        Congratulations, these people want to be your friend!
-                    </td>
+            matchingSection = <h3 id='Matches-congrats'>
+                <p>Congratulations,</p>
+                <p>You got potential pal-ed!</p> 
```

Comment:
Not sure what pal-ed means, maybe change it to congratulations you have a match and based on the number of matches have a different response?